mattermost/mattermost-webapp
# [MM-28323] Don't show old tutorial or what's new modal for cloud instances

### Summary
On first login, users should be brought directly to the next steps view, skipping the what's new modal, and the old style tutorial.

Test steps:
Invite yourself to the spinwick server
login
you should be directed right to the next steps view 

(requires cloud license)


Remove the cloud license
Do the above
You should see the what's new and old style tutorial


#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28323
#### Related Pull Requests
n/a

#### Screenshots
n/a
